src/workerd/api/r2-admin.c++

```diff
@@ -12,10 +12,9 @@
 
 namespace workerd::api::public_beta {
 jsg::Ref<R2Bucket> R2Admin::get(jsg::Lock& js, kj::String bucketName) {
-  KJ_IF_MAYBE(a, adminAccount) {
-    auto& j = KJ_ASSERT_NONNULL(jwt, "adminAccount without corresponding jwt");
+  KJ_IF_MAYBE(j, jwt) {
     return jsg::alloc<R2Bucket>(featureFlags, subrequestChannel, kj::mv(bucketName),
-                                kj::str(*a), kj::str(j), R2Bucket::friend_tag_t{});
+                                kj::str(j), R2Bucket::friend_tag_t{});
```

Comment:
This should be `kj::str(*j)` now.
It's a subtle change, but `j` changed from a reference to a pointer, and without the dereference `kj::str` is going to print the address and not the contents (saw it when running tests locally)

Reply:
Nice catch, I definitely missed this.